src/bucket.ts

```diff
@@ -629,7 +629,10 @@ class Bucket extends ServiceObject {
   acl: Acl;
   iam: Iam;
 
-  getFilesStream: (query?: GetFilesOptions) => Readable;
+  getFilesStream(query?: GetFilesOptions): Readable {
+    // placeholder body, overwritten in constructor
```

Comment:
This seems reasonble to me, but I would open a ticket in parser to fix this issue, and reference the ticket # here.

Reply:
You mean in api-documnter? Yeah, I can open an issue there.

Reply:
done: https://github.com/microsoft/rushstack/issues/3031